src/account.js

```diff
@@ -40,40 +40,35 @@ export const getAccountAddress = async (accountName) => {
   };
 
   let accountAddress;
-  try {
-    const code = await registry.scripts.getAccountAddressTemplate(addressMap);
 
+  const code = await registry.scripts.getAccountAddressTemplate(addressMap);
+
+  const args = [
+    [name, t.String],
+    [managerAddress, t.Address],
+  ];
+
+  const [result] = await executeScript({
+    code,
+    args,
+    service: true,
+  });
+  accountAddress = result;
+
+  if (accountAddress === null) {
+    const code = await registry.transactions.createAccountTemplate(addressMap);
+    const publicKey = await pubFlowKey();
     const args = [
-      [name, t.String],
+      [name, publicKey, t.String],
       [managerAddress, t.Address],
     ];
-    accountAddress = await executeScript({
+    const [result] = await sendTransaction({
```

Comment:
`flow-cadut` shall resolve arguments, so we can simply pass values here

```js
const args = [name, publicKey, managerAddress]
```

Reply:
I went through all instances of `executeScript` and `sendTransaction` to assure that passed args allow flow-cadut to resolve the arg types. Since we don't need fcl/types in these cases I removed that as well.